### ISSUE TITLE
[BUG] Updating from 2.2.20 to 2.2.23 raised a warning

### ISSUE THREAD

**Issue by eltoma-alexkuc** [lucatume/wp-browser#304]
**Environment**
OS: OSX 10.13.6
PHP version: 7.3.8
Installed Codeception version: 3.1.0
Installed wp-browser version: 2.2.20 -> 2.2.23
WordPress version: 5.2.3
Local development environment: Docker (compose file)
WordPress structure and management: default

**Describe the bug**
When performing Composer update/upgrade, I got 2 warnings: `Warning: Ambiguous class resolution` (full warnings follow).

**Output**
```bash
Warning: Ambiguous class resolution, "tad\WPBrowser\Compat\Codeception\Unit" was found in both "<repo>/wp_plugin/vendor/lucatume/wp-browser/src/tad/WPBrowser/Compat/PHPUnit/Version8/Unit.php" and "<repo>/wp_plugin/vendor/lucatume/wp-browser/src/tad/WPBrowser/Compat/PHPUnit/Base/Unit.php", the first will be used.
```

```bash
Warning: Ambiguous class resolution, "tad\WPBrowser\Compat\PHPUnit\Testcase" was found in both "<repo>/wp_plugin/vendor/lucatume/wp-browser/src/tad/WPBrowser/Compat/PHPUnit/Version8/Testcase.php" and "<repo>/wp_plugin/vendor/lucatume/wp-browser/src/tad/WPBrowser/Compat/PHPUnit/Base/Testcase.php", the first will be used.
```

**To Reproduce**
Pre-update `composer.json` file:
```json
{
    "name": "...",
    "description": "...",
    "type": "project",
    "license": "Proprietary",
    "authors": [
        {
            "name": "...",
            "email": "..."
        }
    ],
    "require": {},
    "require-dev": {
      "lucatume/wp-browser": "^2.2.18"
    }
}
```

Then `composer update`.

**Expected behavior**
Class resolution warnings shouldn't arise as (potentially) it could mess up loading of classes

**Comment by lucatume** [comment]
@eltoma-alexkuc thanks for reporting the issue. 
There are indeed two classes, and have been for some time now, but recent changes must be the cause follow that warning. 

I will look into this.

**Comment by lucatume** [comment]
I have found no issues with this when updating without using the `-a` (authoritative class mapping) flag.

Correct me if I'm wrong but the message will only pop-up when using the `-a` flag.

I will look into this nonetheless

**Comment by eltoma-alexkuc** [comment]
@lucatume to be honest I didn't try with `-a` flag as common practise is to update dependencies using just `composer update` (from my experience) unless one of the dependencies would explicitly state that a different update method is required

**Comment by lucatume** [comment]
@eltoma-alexkuc This should be fixed in `2.2.26`; let me know if not.
